### PR TITLE
Fix DNS nameserver env var in tests

### PR DESCRIPTION
**Pull request by masnax**
@simondeziel this should fix your issue

### CODE REVIEW

**Review by simondeziel — APPROVED** [review]
Thanks!

File: microcloud/test/suites/add.sh [hunk 1]
```diff
@@ -123,8 +123,8 @@ test_add_interactive() {
   export IPV4_SUBNET="10.1.123.1/24"
   export IPV4_START="10.1.123.100"
   export IPV4_END="10.1.123.254"
-  export CUSTOM_DNS_ADDRESSES="10.1.123.1,8.8.8.8" # comma-separated list of custom DNS addresses to be set for the OVN uplink.
   export IPV6_SUBNET="fd42:1:1234:1234::1/64"
+  export DNS_ADDRESSES="10.1.123.1,fd42:1:1234:1234::1/64"
```
Comment (roosterfish) [inline microcloud/test/suites/add.sh:127]: Should the IPv6 DNS address have a subnet mask? 
If so, why isn't it required for the IPv4 address?
From what I understood the validation will check if the addresses set in `DNS_ADDRESSES` are actually there.

I have never worked with this test suite so please bare with me :)

**Review by roosterfish — COMMENTED** [review]
Looks good to me there is just one variable I am not sure about.

File: microcloud/test/suites/add.sh (hunk 1 above)
Comment (tomponline) [reply microcloud/test/suites/add.sh:127]: No it shouldnt have a subnet mask

Comment (tomponline) [reply microcloud/test/suites/add.sh:127]: If the tests are passing with this value then the system isnt performing sufficient validation.

Comment (masnax) [reply microcloud/test/suites/add.sh:127]: That's what #277 is for.

Comment (masnax) [reply microcloud/test/suites/add.sh:127]: Though if the tests actually did pass that means LXD also isn't validating this config key when it receives it.

Comment (tomponline) [reply microcloud/test/suites/add.sh:127]: Possibly, both are not doing it correctly, can you check and update where needed please.

Comment (simondeziel) [reply microcloud/test/suites/add.sh:127]: The microcloud tests are failing on this unexpected CIDR notation, see https://github.com/canonical/microcloud/actions/runs/8711674992/job/23896307777?pr=281#step:7:2774:

```
---
10.1.123.1/24                                         # setup ipv4/ipv6 gateways and ranges
10.1.123.100
10.1.123.254
fd42:1:1234:1234::1/64
10.1.123.1,fd42:1:1234:1234::1/64
+ sed -e /^\s*#/d -e /^\s*$/d
Invalid input: Item "fd42:1:1234:1234::1/64": Not an IP address "fd42:1:1234:1234::1/64"
```

Comment (simondeziel) [reply microcloud/test/suites/add.sh:127]: Dropping that bogus `/64` makes the tests go much further in #281

**Review by simondeziel — APPROVED** [review]

**Review by tomponline — APPROVED** [review]

**Pull request merged**